src/components/interactive/Avatar.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+export const VALID_BG_COLORS = [
+  'white',
+  'haze',
+  'lightGray',
+  'orange',
+  'charcoal',
+  'navy',
+  'aqua',
+  'gray',
+  'red',
+  'purple',
+];
+
+const NON_INVERTED_BG_COLORS = ['white', 'haze', 'lightGray'];
```

Comment:
Not a blocker, but a minor stylistic tip.

Some colors belong to both lists, which can be prone to error if someone modifies one of these lists later without modifying the other. One potential improvement is to make each color belong only to one list:

```
export const VALID_LIGHT_COLORS = [
	'white',
	'haze',
	'lightGray',
];

export const VALID_DARK_COLORS = [
	'gray',
	'charcoal',
	'orange',
	'red',
	'purple',
	'aqua',
	'navy',
];
```

For prop types, you can spread them:

```
	bgColor: PropTypes.oneOf([
		...VALID_LIGHT_COLORS,
		...VALID_DARK_COLORS,
	]),
```

And you can check the dark colors list directly in your `cx()`, similar to what you're already doing:

```
{ 'inverted': VALID_DARK_COLORS.includes(bgColor) }
```